concent-builder/configure-jenkins.yml

```diff
@@ -29,28 +29,18 @@
             state: directory
           with_items:
             - concent-secrets
-            - concent-secrets/{{ cluster }}/
-            - concent-secrets/cloud/
             - concent-secrets/jenkins/
 
     - become:      yes
       become_user: root
       block:
         - name: Upload secrets
           copy:
-            src:   "{{ local_secret_dir }}/{{ cluster }}/{{ item }}"
-            dest:  "{{ jenkins_home_dir }}/concent-secrets/{{ cluster }}/{{ item }}"
+            src:   "{{ local_secret_dir }}/jenkins/.token"
+            dest:  "{{ jenkins_home_dir }}/concent-secrets/jenkins/.token"
```

Comment:
Why isn't the token encrypted with Ansible Vault?

Reply:
This is not a part of this pull request. It can be encrypted. It's used to install jenkins plugins in the same playbook. The best solution is to upload it temporary when playbook is running.

Reply:
Then it should have been done either in #354 or #297 (whichever came last).

This needs to be fixed, either here or in a separate pull request.